README.md

```diff
@@ -47,7 +47,7 @@ var test = goiabada.test.bind( goiabada );
 
 #### Assertions:
 
-- `ok( value[, message ] )`: pass when the value is truthy.
+- `ok( value[, message ] )`: pass when the value is true.
```

Comment:
`ok` evaluates the given value to be equivalent to the Boolean true, in a non strict comparison. E.g.: a non empty string will pass.